packages/ui/src/visually-hidden/style.module.css

```
@@ -0,0 +1,16 @@
+@layer wp-ui-utilities, wp-ui-components, wp-ui-compositions, wp-ui-overrides;
+
+@layer wp-ui-components {
+	.visually-hidden {
```

I've seen example of a selector including 


```suggestion
	.visually-hidden.not(:focus):not(:active)
```

Should we consider the amend?

---

Interesting. Initially I thought it was for skip links, but it's more of a defensive trick for misuse? I could go either way on this, but I'm leaning a bit towards not adding it, because I did an audit of the codebase right now and I didn't find any instances of this type of misuse. I wouldn't object if you want to try it though.

---

I also interpreted it as something useful for skip links (or any visually hidden, tabbable/focusable element) — and so does [the a11y collective article](https://www.a11y-collective.com/blog/visually-hidden/#h-making-hidden-content-keyboard-accessible) and [josh comeau](https://www.joshwcomeau.com/snippets/react-components/visually-hidden/#:~:text=Not%20for%20interactive%20elements).

I thought it would be a valuable feature, but no strong opinions in case you prefer to keep the current selector

---

Let's keep this in our back pocket for when we discover a use case 👍